static/panes/compiler.ts

```diff
@@ -1139,7 +1140,7 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         if (filters.libraryCode && !this.compiler.supportsLibraryCodeFilter) {
             delete filters.libraryCode;
         }
-        (this.compiler as any).disabledFilters.forEach(filter => {
+        (this.compiler.disabledFilters as string[]).forEach(filter => {
```

Comment:
This is one cast I couldn't get rid of, `CompilerInfo.disabledFilters` can be a string briefly but lib/base-compiler splits it to an array immediately